test/unit/notched-outline/index.test.tsx

```diff
@@ -3,29 +3,31 @@ import {assert} from 'chai';
 import * as td from 'testdouble';
 import {mount, shallow} from 'enzyme';
 import NotchedOutline from '../../../packages/notched-outline/index';
+import {MDCNotchedOutlineAdapter} from '@material/notched-outline/adapter';
 
 suite('NotchedOutline');
 
+function getAdapter(foundation: any): MDCNotchedOutlineAdapter {
+  // @ts-ignore
```

Comment:
what are you ignoring here. please comment.

Reply:
adapter_ method is protected in mdc-web, that's how I override it

Reply:
gotcha - please comment that inline with the ignore. Thanks!